website/content/docs/k8s/helm.mdx

```diff
@@ -169,16 +169,35 @@ Use these links to navigate to a particular top-level stanza.
         - gossip encryption key defined by `global.gossipEncryption.secretName`.
         To discover the service account name of the Consul client, run
             ```shell-session
-            $ helm template --show-only templates/client-serviceaccount.yaml <release-name> charts/consul
+            $ helm template --show-only templates/client-serviceaccount.yaml <release-name> hashicorp/consul
```

Comment:
This is not rendering correctly. You might have to play with the indentation.

![image](https://user-images.githubusercontent.com/29551334/161106033-178b6adc-f6ee-4d2d-9b90-0ad286b73c12.png)

Reply:
How are you able to preview this?  I would love to also be able to do this myself.

Reply:
I added a comment in the PR that explains this. But yeah, it's still breaking 😢

Reply:
Maybe it needs a newline before and after? or maybe it's wrong indentation?

Reply:
@karl-cardenas-coding can you elaborate exactly what should happen here in terms of the visual we should see? When I look at this [preview](https://consul-d50kyx45z-hashicorp.vercel.app/docs/k8s/helm), all code blocks match the indentation of the lines above them and this one seems to as well.  Am I trying to widen this code block?

Reply:
oh sorry.  I thought the indentation was wrong.  I see it now.  :face-palm.
the markdown is showing up in the rendered code block.